components/HomeComponents/contact-us.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { PageType } from '../Types'
+import ContactData from '../dataComponents/contactdata'
+import { Button } from '../dataComponents/button'
+import SignupForm from '../form/signUpForm'
+
+export default function ContactUs({ data }: PageType) {
+  return (
+    <>
+      <section className=" mt-32 md:mt-64 ">
+        <div
+          id="contactUsItems"
+          className="grid md:grid-cols-2 place-items-center bg-[#F4F3FD] md:h-[600px] relative"
+        >
+          <div className="grid">
+            <div className="w-8/12 p-2 md:p-0">
+              <Button
+                className={
+                  'text-xl md:text-3xl bg-[#FFFFFF] text-[#4E47F9] px-8 py-2 rounded-full mb-5 font-bold'
```

Comment:
Why not using bg-light from the tailwind config? The whole idea is to not duplicate values. please change it everywhere.
Also for the rest of the colors. define a variable for them in the config and use it.